rust/abacus-base/src/metrics/core.rs

```diff
@@ -191,19 +197,30 @@ impl CoreMetrics {
             outbox_state,
             latest_checkpoint,
 
+            json_rpc_client_metrics: OnceCell::new(),
```

Comment:
This was partially to maintain the same methodology as the other, but also partially because it allows for lazy init, and in our case we actually don't need them at all if it is not using a quorum provider so it won't be created in that case. Small optimization at best. It really would not be a big deal if we shoved both into the "new" function.